diff -Nurpa -x '*~' -x '*.orig' -x '*.rej' -x '*.swp' vdradmin-am-3.6.7//vdradmind.pl newdir/vdradmind.pl
--- vdradmin-am-3.6.7//vdradmind.pl	2010-07-20 05:05:50.790479567 +0300
+++ vdradmin-am-3.6.7/vdradmind.pl	2010-07-20 05:06:08.861416117 +0300
@@ -517,8 +517,6 @@ if (-e "$PIDFILE") {
             exit(1);
         }
         Log(LOG_ERROR, "The pid $pid is not a running $PROCNAME process, so I'll start anyway.\n");
-    } else {
-        Log(LOG_ERROR, "$PIDFILE exists, but is empty or contains bogus data, so I'll start anyway.\n");
     }
 }
 
@@ -760,7 +758,6 @@ sub check_permissions {
     check_rw_dir($ETCDIR) or $rc = 0;
     check_rw_dir($CERTSDIR) if ($UseSSL);
     check_rw_dir($TEMPLATECACHE) or $rc = 0;
-    check_rw_file($PIDFILE) or $rc = 0;
     check_rw_file($CONFFILE) or $rc = 0;
 
     if ($CONFIG{AT_FUNC} || $FEATURES{AUTOTIMER}) {
@@ -3559,7 +3556,6 @@
     chomp(my $pid = <PID> || "");
     close(PID) or Log(LOG_WARNING, "Error closing pid file '$pidfile': $!");
     if ($pid !~ /^\d+$/) {
-        Log(LOG_WARNING, "Ignoring bogus process id '$pid' in pid file '$pidfile'");
         $pid = undef;
     }
     return $pid;

