Index: vdradmin-am-3.6.7/vdradmind.pl
===================================================================
--- vdradmin-am-3.6.7/vdradmind.pl
+++ vdradmin-am-3.6.7/vdradmind.pl	2010-07-20 04:56:32.993478699 +0300
@@ -759,7 +759,6 @@
     check_rw_dir($ETCDIR) or $rc = 0;
     check_rw_dir($CERTSDIR) if ($UseSSL);
     check_rw_dir($TEMPLATECACHE) or $rc = 0;
-    check_rw_dir($LOGDIR) or $rc = 0;
     check_rw_file($PIDFILE) or $rc = 0;
     check_rw_file($CONFFILE) or $rc = 0;
 
